docs/misc/analytics.md

```diff
@@ -122,6 +122,39 @@ Add event to the analytics overview in the [company Notion](https://www.notion.s
 
 ## Changelog
 
+### 1.17
+
+Added:
+
+-   create-backup
+    -   status: 'finished' | 'error'
+    -   error: string
+-   app-update
+    -   fromVersion: string
+    -   toVersion: string
+    -   status: 'finished' | 'cancelled' | 'error'
+    -   version: 'stable' | 'beta'
+-   suite-ready
+    -   customBackends: ['btc', 'eth', 'ada', ...] (array of coins with custom backend)
+    -   autodetectLanguage: boolean;
+    -   autodetectTheme: boolean;
+-   settings/coin-backend
+    -   symbol: 'btc', 'eth', 'ada', ...
+    -   type: 'default' | 'blockbook' | 'electrum' | 'ripple' | 'blockfrost'
+    -   totalRegular: number
+    -   totalOnion: number
```

Comment:
just an idea - what about to add Tor status here and some tracking for DisableTor modal? But probably later...

Reply:
`tor` status can be received from `suite-ready` or from `settings/toggle-tor` but sure we can discuss it and improve later